src/pubkey.cpp

```diff
@@ -5,9 +5,17 @@
 
 #include <pubkey.h>
 
+#include <hash.h>
 #include <secp256k1.h>
+#include <secp256k1_extrakeys.h>
 #include <secp256k1_recovery.h>
 #include <secp256k1_schnorrsig.h>
+#include <span.h>
+#include <uint256.h>
+
+#include <algorithm>
+
+#include <assert.h>
```

Comment:
```suggestion
#include <cassert>
#include <algorithm>
```

(group all standard library imports together and prefer the c++ versions of the c headers).

Reply:
Done. Yep that was a question I was slightly mulling over. I'll take a look and see if i see the styling docs (if any).